packages/webpack/src/config/base.js

```diff
@@ -422,12 +425,7 @@ export default class WebpackBaseConfig {
 
   warningFixFilter() {
     const { buildOptions, options: { _typescript = {} } } = this.buildContext
-    const filters = [
-      // Hide warnings about plugins without a default export (#1179)
-      warn => warn.name === 'ModuleDependencyWarning' &&
-        warn.message.includes(`export 'default'`) &&
-        warn.message.includes('nuxt_plugin_')
-    ]
+    const filters = buildOptions.warningFixFilters || []
```

Comment:
Actually, I've used a really internal name when creating this plugin. For exposing I suggest a better name like `warningIgnoreFilters` (for consistency we can also rename internal plugin name)

PS: `buildOptions` is same as `options` (https://github.com/nuxt/nuxt.js/blob/v2.8.0/packages/builder/src/context/build.js#L10) -- we can try to make usage more consistent in config refactor.

Reply:
Alright, will update that.